ui/shared/components/panel/variants/Transcripts.jsx

```diff
@@ -42,7 +94,62 @@ const TRANSCRIPT_LABELS = [
   },
 ]
 
-const Transcripts = React.memo(({ variant, genesById, transcriptsById, updateMainTranscript, project }) => (
+const transcriptIdDetails = (transcript, variant, { transcriptsById, project, updateMainTranscript }) => (
+  <div>
+    {transcriptsById[transcript.transcriptId]?.refseqId && (
+      <div>
+        <HeaderLabel>RefSeq:</HeaderLabel>
+        <a
+          href={`https://www.ncbi.nlm.nih.gov/nuccore/${transcriptsById[transcript.transcriptId].refseqId}`}
+          target="_blank"
+          rel="noreferrer"
+        >
+          {transcriptsById[transcript.transcriptId].refseqId}
+        </a>
+      </div>
+    )}
+    {TRANSCRIPT_LABELS.map(({ shouldShow, ...labelProps }) => (
+      shouldShow(transcript, transcriptsById) && (
+        <Label key={labelProps.content} size="small" horizontal {...labelProps} />
+      )
+    ))}
+    {
+      variant.variantGuid && project?.canEdit && (
+        <span>
+          <VerticalSpacer height={5} />
+          {
+            transcript.transcriptId === variant.selectedMainTranscriptId ?
+              <Label content="User Chosen Transcript" color="purple" size="small" /> : (
+                <DispatchRequestButton
+                  onSubmit={updateMainTranscript(transcript.transcriptId)}
+                  confirmDialog="Are you sure want to update the main transcript for this variant?"
+                >
+                  <Label as="a" content="Use as Main Transcript" color="violet" basic size="small" />
+                </DispatchRequestButton>
+              )
+          }
+        </span>
+      )
+    }
+  </div>
+)
+
+const ANNOTATION_SECTIONS = [
+  [{ title: 'Codons' }, { title: 'Amino Acids' }],
+  [
+    { title: 'Biotype' },
+    {
+      title: 'Intron/Exon',
+      getContent: c => ['intron', 'exon'].filter(f => c[f]).map(f => `${camelcaseToTitlecase(f)} ${c[f].index}/${c[f].total}`).join(', '),
```

Comment:
this is the only actual change to the Transcripts content, everything else is just a refactor to share code.  We use to show `cdnaPosition` here, but that has been absent from the data for a while so theres no reason to keep it